model/riscv_pmp_regs.sail

```diff
@@ -158,15 +167,31 @@ function pmpReadCfgReg(n) = {
 }
 
 /* Helpers to handle locked entries */
-function pmpLocked(cfg: Pmpcfg_ent) -> bool =
-   cfg.L() == 0b1
+function pmpLockBit(cfg: Pmpcfg_ent) -> bool = cfg.L() == 0b1
```

Comment:
Maybe split the renaming of these functions into a separate commit?

Reply:
I needed that here in the Smepmp implementation. It is not simply renaming as explained in this [comment](https://github.com/riscv/sail-riscv/pull/196#discussion_r1280540596)